src/configuration.ts

```diff
@@ -1,15 +1,115 @@
 import { NotionConfig } from './service/notion/config';
+import { FormConfig, TypeField } from './types/form';
+import { CollapsedFormConfig } from './types/collapsedForm';
 
 /**
  * Notion configuration for testing
  */
-export const exampleConfiguration: NotionConfig = {
+export const exampleConfiguration: {
+  notion: NotionConfig;
+  collapsedForm: CollapsedFormConfig;
+  form: FormConfig;
+} = {
   /**
-   * Notion database id
+   * API configuration
    */
-  databaseId: '',
+  notion: {
+    /**
+     * Notion database id
+     */
+    databaseId: '',
+    /**
+     * Client secret from integration
+     */
+    clientSecret: '',
+  },
   /**
-   * Client secret from integration
+   * Configuration for collapsed form
    */
-  clientSecret: '',
+  collapsedForm: {
+    title: 'Using Editor.js?',
+    description: 'Take a 2-minutes survey🙏',
+  },
+  /**
+   * Configuration for full form
+   */
+  form: {
+    description:
+      'We want to better understand your goals and requirements. Please, provide us some insights.',
+    items: [
+      {
+        label: 'Product type',
+        field: {
+          name: 'product_type',
+          type: TypeField.Select,
+          options: [
+            {
+              name: 'B2B',
+              value: 'B2B',
+            },
+            {
+              name: 'B2C',
+              value: 'B2C',
+            },
+          ],
+        },
+      },
+      {
+        label: 'Your product size',
+        field: {
+          type: TypeField.Select,
+          name: 'product_size',
+          options: [
+            {
+              name: '<100 DAU',
+              value: '<100 DAU',
+            },
+            {
+              name: '100 DAU - 1000 DAU',
+              value: '100 DAU - 1000 DAU',
+            },
+            {
+              name: '> 1000 DAU',
+              value: '> 1000 DAU',
+            },
+          ],
+        },
+      },
+      {
+        label: 'Your software license',
+        field: {
+          name: 'software_license',
+          type: TypeField.Select,
+          options: [
+            {
+              name: 'Freeware / Open-Source',
+              value: 'Freeware / Open-Source',
+            },
+            {
+              name: 'Permissive',
+              value: 'Permissive',
+            },
+            {
+              name: 'Proprietary',
+              value: 'Proprietary',
+            },
+          ],
+        },
+      },
+      {
+        label: 'Your suggestions to the Editor.js',
+        field: {
+          name: 'suggestions',
+          type: TypeField.Textarea,
+          placeholder: 'Enter your ideas, requests or issues',
+        },
+      },
+      {
+        field: {
+          type: TypeField.Submit,
+          textContent: 'Submit',
+        },
+      },
```

Comment:
I think the Submit shouldn't be in a config. I'll be always added to the end of the form.